Casks/a/archi.rb

```diff
@@ -0,0 +1,32 @@
+cask "archi" do
+  arch arm: "-Silicon", intel: ""
```

Comment:
```suggestion
  arch arm: "-Silicon"
```

The stanza defaults to an empty string.